src/cc65/symtab.c

```diff
@@ -1109,7 +1109,7 @@ void EmitDebugInfo (void)
         }
         Sym = SymTab->SymHead;
         while (Sym) {
-            if ((Sym->Flags & (SC_CONST|SC_TYPE)) == 0) {
+            if ((Sym->Flags & (SC_CONST|SC_TYPEMASK)) == 0) {
```

Comment:
It seems there should have been blanks here before, shouldn't they?

Reply:
Fixed.